### PR TITLE
fix dataset location generation

### PR DESCRIPTION
**Pull request by pbruski**

### CODE REVIEW

File: src/main/kotlin/com/atlassian/performance/tools/infrastructure/dataset/HttpDatasetPackage.kt [hunk 1]
```diff
@@ -53,7 +53,7 @@ internal class HttpDatasetPackage(
         val dataset = (newFiles.singleOrNull()
             ?: throw Exception("Expected one new folder. Found $newFiles."))
 
-        return "$destination/$dataset"
+        return ssh.execute("cd '$destination/$dataset' && pwd").output.trimEnd('\n')
```
Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/infrastructure/dataset/HttpDatasetPackage.kt:56]: Huh, interesting. What was the symptom without this fix?

Comment (dagguh) [inline src/main/kotlin/com/atlassian/performance/tools/infrastructure/dataset/HttpDatasetPackage.kt:56]: Changelog please

Comment (pbruski) [reply src/main/kotlin/com/atlassian/performance/tools/infrastructure/dataset/HttpDatasetPackage.kt:56]: It clashed with a recent change that Konrad did - it blew up during Docker startup.

**Comment by mzyromski-atlassian** [comment]
Released in `4.15.1`.

**Pull request merged**